pkg/adapter/v2/config.go

```diff
@@ -126,6 +137,14 @@ func (e *EnvConfig) GetName() string {
 	return e.Name
 }
 
+func (e *EnvConfig) SetupTracing(logger *zap.SugaredLogger) error {
+	config, err := tracingconfig.JsonToTracingConfig(e.TracingConfigJson)
+	if err != nil {
+		logger.Error("Tracing configuration is invalid, using the no-op default", zap.Error(err))
```

Comment:
I see this message, but I don't see `config = defaultNoopTracingConfig`. If this is hidden behind `tracingconfig.JsonToTracingConfig`, I believe the method should be renamed to reflect the intention, or the API should be improved to avoid obfuscation the defaulting.

Reply:
For now it's godoc-ed that the method returns a default no-op configuration and I wanted to keep the consistency with the names of metrics and logging config. Since https://github.com/knative/pkg/pull/1228 is already merged, if for you is fine, we can change this in a followup

Reply:
Sure that's fine for me, just wanted to point it out.